### DISCUSSION TITLE
epoptes not showing booted clients

### DISCUSSION THREAD

**Discussion by danboid** [ltsp/ltsp#864] [Q&A]
I was hoping to use epoptes to do a network check earlier but then I discovered epoptes wasn't showing any of my clients as being booted when I could see at least one machine was fully booted into LTSP.

What ports or network settings are required for epoptes to see clients and do a connection check? It seems odd to me that the client is booting but epoptes isn't showing any machines.

My LTSP server is running Ubuntu 22.04

Thanks

**Comment by alkisg** [comment]
For epoptes discussions it's best to use https://github.com/epoptes/epoptes/discussions.

First try to temporarily disable your firewall. If Epoptes works, then it's a firewall issue, otherwise it's not.
The latest Epoptes version available from the proposed PPA is firewall friendly, the previous versions aren't.
